pr-data.csv

```diff
@@ -2383,7 +2383,7 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/configuration,org.apache.ignite.internal.configuration.util.ConfigurationUtilTest.testSchemaFields,ID,MovedToGradle,,https://github.com/apache/ignite-3/commit/c91f39cd80dfd143e5ac848486366ad6ed124ce5
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/page-memory,org.apache.ignite.internal.pagememory.freelist.AbstractFreeListTest.testMultiThread,ID,MovedToGradle,,https://github.com/apache/ignite-3/commit/c91f39cd80dfd143e5ac848486366ad6ed124ce5
 https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgs,ID,,,
-https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgsNullConfig,ID,,,
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/runner,org.apache.ignite.internal.runner.app.IgniteCliRunnerTest.smokeTestArgsNullConfig,ID,Deleted,,https://github.com/apache/ignite-3/commit/dd6ffb555b99d23b1484083e50831f934ee20405
```

Comment:
Skimming over https://github.com/apache/ignite-3/commit/dd6ffb555b99d23b1484083e50831f934ee20405 shows no changes in `IgniteCliRunner...`. What do you get with `git show --name-status dd6ffb555b99d23b1484083e50831f934ee20405 | grep IgniteCliRunner`.

Reply:
Sorry I forgot to mention this, they changed the name of the test file to IgniteRunnerTest.java in some commits before. I thought it was more appropriate to list the delete in this commit where you'll see the `smokeTestArgsNullConfig` is deleted.

Reply:
Let's list both, first some `MovedOrRenamed` for the test class and then later `Deleted` for the test method.

Reply:
Do I make another entry in the csv with just the test class? With empty test method?

Reply:
If the test class had the reported test method when the test class was moved, then keep the same class name and method name, just add `MovedOrRenamed` in the status (and the URL to the commit in `Notes`).